components/script/script_thread.rs

```diff
@@ -1657,17 +1657,23 @@ impl ScriptThread {
                 document.react_to_environment_changes()
             }
 
-            // Update animations and send events.
-            self.update_animations_and_send_events(can_gc);
+            // > 11. For each doc of docs, update animations and send events for doc, passing
+            // > in relative high resolution time given frameTimestamp and doc's relevant
+            // > global object as the timestamp [WEBANIMATIONS]
+            document.update_animations_and_send_events(can_gc);
```

Comment:
Btw does this not deal with the ordering problem encountered and mentioned at https://github.com/servo/servo/pull/34486?

Reply:
No, unfortunately not, because the issue is both ordering and skipping non-top-level iframes that are not children of top-level iframes in the same ScriptThread. Those iframes never have their rendering updated at all currently.